exporter/uptraceexporter/exporter.go

```diff
@@ -119,7 +99,31 @@ func (e *traceExporter) pushTraceData(ctx context.Context, traces pdata.Traces)
 	return nil
 }
 
-func (e *traceExporter) Shutdown(ctx context.Context) error {
+func (e *traceExporter) start(_ context.Context, _ component.Host) error {
+	if e.cfg.HTTPClientSettings.Endpoint != "" {
+		e.logger.Warn("uptraceexporter: endpoint is not supported; use dsn instead")
+	}
```

Comment:
This should be part of the validation, probably a separate PR.

Reply:
Agreed. I just moved the original logic.